### PR TITLE
Update the portable Linux and Windows pypy builds

### PR DESCRIPTION
**Pull request by NeonFlames**

### CODE REVIEW

**Review by JaskRendix — APPROVED** [review]

File: buildconfig/build_pypy_portable_linux.sh [hunk 1]
```diff
@@ -1,33 +1,49 @@
...
-[ -e README.rst ] && mv README.rst docs
-[ -e LICENSE ] && mv LICENSE docs/PYPY_LICENSE
-cp ../../../LICENSE docs/TUXEMON_LICENSE
-cp ../../../run_tuxemon.py .
-cp ../../../CONTRIBUTING.md .
-cp ../../../CREDITS.md .
-cp ../../../README.md .
-cp ../../../SPYDER_README.md .
+
+apt install -y curl coreutils
```
Comment (bitcraft) [inline buildconfig/build_pypy_portable_linux.sh]: please remove `-y`.  we should not make a habit of building as root

File: buildconfig/build_pypy_portable_linux.sh [hunk 2]
```diff
@@ -1,33 +1,49 @@
...
-pypy_archive="$pypy_build.tar.bz2"
-mkdir -p build/pypy-linux64
-cd build/pypy-linux64
-[ -e $pypy_archive ] || wget -q https://downloads.python.org/pypy/$pypy_archive
-[ -e $build_folder ] || {
- tar xf $pypy_archive
- mv $pypy_build $build_folder
+
+#############  NOTICE  ##############
+# Tested in a debian stable container
```
Comment (bitcraft) [inline buildconfig/build_pypy_portable_linux.sh]: please include debian version

File: buildconfig/build_pypy_portable_windows.sh [hunk 3]
```diff
@@ -1,39 +1,53 @@
...
-pypy_archive="$pypy_build.zip"
-mkdir -p build/pypy-win32
-cd build/pypy-win32
-[ -e $pypy_archive ] || wget -q https://downloads.python.org/pypy/$pypy_archive
-[ -e $build_folder ] || {
- unzip $pypy_archive
- mv $pypy_build $build_folder
+
+#############  NOTICE  ##############
+# Tested in a debian stable container
```
Comment (bitcraft) [inline buildconfig/build_pypy_portable_windows.sh]: please include the version

File: buildconfig/build_pypy_portable_windows.sh [hunk 4]
```diff
@@ -1,39 +1,53 @@
...
-[ -e README.rst ] && mv README.rst docs
-[ -e LICENSE ] && mv LICENSE docs/PYPY_LICENSE
-cp ../../../LICENSE docs/TUXEMON_LICENSE
-cp ../../../run_tuxemon.py .
-cp ../../../CONTRIBUTING.md .
-cp ../../../CREDITS.md .
-cp ../../../README.md .
-cp ../../../SPYDER_README.md .
+
+apt install -y wine curl coreutils p7zip-full
```
Comment (bitcraft) [inline buildconfig/build_pypy_portable_windows.sh]: please remove `-y`

**Review by bitcraft — CHANGES_REQUESTED** [review]
looks good, minor changes requested

**Review by bitcraft — APPROVED** [review]

**Pull request merged**